DataFormats/HGCalReco/interface/Trackster.h

```diff
@@ -6,8 +6,10 @@
 
 #include <array>
 #include <vector>
+
 #include "DataFormats/Provenance/interface/ProductID.h"
 #include "DataFormats/Math/interface/Vector3D.h"
+#include "CommonTools/RecoAlgos/interface/MultiVectorManager.h"
```

Comment:
A DataFormat package may not depend on `CommonTools/RecoAlgos`, see https://github.com/cms-sw/cmssw/issues/48822